### PR TITLE
Fix forbidden null sending as attribute value

### PR DESCRIPTION
**Pull request by orzechdev**
I want to merge this change because... it fixes forbidden null sending as an attribute value.

**PR intended to be tested with API branch:** 

### Pull Request Checklist

1. [ ] This code contains UI changes
2. [ ] All visible strings are translated with proper context including data-formatting
3. [ ] Attributes `[data-test-id]` are added for new elements
4. [ ] Changes are mentioned in the changelog
5. [ ] The changes are tested in different browsers and in light/dark mode

### Test environment config

API_URI=https://qa.staging.saleor.cloud/graphql/

### CODE REVIEW

**Comment by patrys** [comment]
Task linked: [SALEOR-3700 Dashboard sends [null] as input where null is forbidden](https://app.clickup.com/t/2549495/SALEOR-3700)

File: schema.graphql [hunk 1]
```diff
@@ -512,6 +512,7 @@ input AttributeFilterInput {
   type: AttributeTypeEnum
   inCollection: ID
   inCategory: ID
+  channel: String
```
Comment (jwm0) [inline schema.graphql:515]: Update the schema to latest version

File: src/attributes/utils/handlers.ts [hunk 2]
```diff
@@ -221,7 +221,10 @@ function getBooleanInput(attribute: AttributeInput) {
 function getDefaultInput(attribute: AttributeInput) {
   return {
     id: attribute.id,
-    values: attribute.value[0] === "" ? [] : attribute.value
+    values:
+      attribute.value[0] === "" || attribute.value[0] === undefined
```
Comment (jwm0) [inline src/attributes/utils/handlers.ts:225]: How about a null value?

**Review by jwm0 — CHANGES_REQUESTED** [review]

File: src/attributes/utils/handlers.ts (hunk 2 above)
Comment (orzechdev) [reply src/attributes/utils/handlers.ts]: Sure, also added

**Review by jwm0 — APPROVED** [review]

**Review by eCzerniak — APPROVED** [review]

**Pull request merged**